app/portainer/services/api/extensionService.js

```diff
@@ -12,19 +12,27 @@ angular.module('portainer.app')
     RBAC: 3
   });
 
-  service.enable = function(license) {
+  service.enable = enable;
+  service.update = update;
+  service.delete = _delete;
+  service.extensions = extensions;
+  service.extension = extension;
+  service.extensionEnabled = extensionEnabled;
+  service.retrieveAndSaveEnabledExtensions = retrieveAndSaveEnabledExtensions;
+
+  function enable(license) {
     return Extension.create({ license: license }).$promise;
-  };
+  }
 
-  service.update = function(id, version) {
+  function update(id, version) {
     return Extension.update({ id: id, version: version }).$promise;
-  };
+  }
 
-  service.delete = function(id) {
+  function _delete(id) {
```

Comment:
_delete?

Reply:
As `delete` is a javascript keyword, eslint doesn't accept functions named `delete`, forced to name them _delete.

Function is still referenced to `delete` inside the service (see L#17).